modules/installation-vsphere-config-yaml.adoc

```diff
@@ -42,8 +42,6 @@ networking:
 ---
```

Comment:
Up above: There should be a content type before the primary title in Line 15.

Reply:
Sorry. I'm only seeing this now. I'll make this update through another open PR. This is clearly a legacy docs update.